### DISCUSSION TITLE
Sensor description - UPS Power, Output Power, Internal/External (CTx) Power...

### DISCUSSION THREAD

**Discussion by babajun12** [davidrapan/ha-solarman#222] [Q&A] [answered]
Hi!
Very nice HA integration with lots of interesting information/entities.
Is there any documentation what the power sensors stands for?

Grid Power - measured at the CT clamps
Load Power - Calculated Power of grid and Battery ?
UPS Power - ?
Output Power - Inverter Power?
Internal/External Power - ?
Power loss is calculated between GridPower and LoadPower?

Thanks!

**Comment by davidrapan** [comment, marked as answer]
* Grid Power is coming from the Grid...
* Load Power is what your household/workplace is drawing
* Load UPS Power shows how much your backup circuit draws during grid cut off
* Output Power is how much power is coming from the device itself
* External Power is far side of the Grid and usually equal to the Grid Power
* Internal Power is close side of the Grid and means devices input from the Grid
* Power losses are calculated as sum of all PV/DC inputs and Battery Power minus Output Power
